rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ ] Hey there! 😄 Since I suggested some changes in setting up associations in both the Book and Person classes, let's make a few tweaks here too to ensure your project runs smoothly. 🛠️
- [ ] The reason for these tweaks is to harmonize the relationships between objects in your project. By fine-tuning how your classes interact and associate with each other, you're essentially creating a more efficient and cohesive system. This means you'll be able to perform operations, retrieve data, and manage your project in a way that aligns with best practices. 🌟 So, these adjustments are like tuning up the engine of a well-oiled machine, ensuring everything runs as expected and delivers great results! 🚀🛠️👏

```suggestion
    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
```